test/e2e/e2e.bats

```diff
@@ -2,25 +2,32 @@
 
 source test/e2e/helpers.sh
```

Comment:
In `helpers.sh` we define a function named `fail` which [collides with bats-core](https://github.com/bats-core/bats-core/blob/8e9f35aa794e38cef6cc9da21003730cce00c3bb/libexec/bats-core/bats-format-pretty#L87). I think the best course of action is to rename it, or maybe removing it completely.

Reply:
updated in current push